### PR TITLE
Release 6.0.0

### PR DESCRIPTION
**Pull request by atugushev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
+# 6.0.0 (2020-03-12)
+
+Backwards Incompatible Changes:
+
+- Remove support for EOL Python 3.5 and 2.7
+  ([#1243](https://github.com/jazzband/pip-tools/pull/1243)). Thanks @jdufresne
+- Remove long deprecated cache dir `~/.pip-tools`
+  ([#1225](https://github.com/jazzband/pip-tools/pull/1225)). Thanks @jdufresne
```
Comment (atugushev) [inline CHANGELOG.md:8]: @jdufresne what do you think, should we add this to the changelog?

Comment (jdufresne) [reply CHANGELOG.md:8]: Probably doesn't matter too much either way. Worse case scenario, the first pip-tools command takes a bit longer and users don't think about it after that. 

No strong preference here. Up to you.

**Review by jdufresne — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (atugushev) [reply CHANGELOG.md]: Ok, since it's not a user-facing change I'll remove it.

**Comment by atugushev** [comment]
Thanks @jdufresne 🙏🏼

**Pull request merged**